Changelog.md

```diff
@@ -1,5 +1,9 @@
 # FOSSA CLI Changelog
 
+## Unreleased
+
+- Allow path filtering when doing cli-side license scans ([#1128](https://github.com/fossas/fossa-cli/pull/1128))
```

Comment:
[very tiny] [minor] we should include scope - `scope:description`, maybe `VendorDependency` or `fossa.yml`?

Reply:
Done!